apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabTheorycraft/ArtifactSubCard/index.tsx

```diff
@@ -87,7 +87,10 @@ export function ArtifactSubCard({
           <BootstrapTooltip
             title={
               <Typography>
-                {t('tabTheorycraft.maxTotalRolls', { value: maxTotalRolls })}
+                <Trans t={t} i18nKey={'tabTheorycraft.maxTotalRolls'}>
+                  {'The current build can only have a maximum of '}
+                  <strong>{{ value: maxTotalRolls } as any}</strong> rolls.
```

Comment:
nit. we dont really need to change this now, but if we ever disallow explicit any.
```suggestion
                  <strong>{{ value: maxTotalRolls } as TransObject}</strong> rolls.
```